cmd/snap-bootstrap/cmd_initramfs_mounts.go

```diff
@@ -326,7 +326,7 @@ func generateMountsModeRun() error {
 	}
 	if !isDataMounted {
 		name := filepath.Base(boot.InitramfsUbuntuDataDir)
-		device, err := unlockIfEncrypted(name)
+		device, err := unlockIfEncrypted(name, true)
```

Comment:
I have a bit of a pet-peeve with bools in function args. It hardly gives any information what the arg means. I much prefer some sort of naming. Anyway, can be fixed later but in the meantime maybe:
```
                last := false
 		device, err := unlockIfEncrypted(name, last)
```